airbyte-commons/src/main/java/io/airbyte/commons/set/MoreSets.java

```diff
@@ -19,4 +20,19 @@
         Sets.difference(set1, set2), Sets.difference(set2, set1)));
   }
 
+  /**
+   * Remove subtract all of the records in one set from another.
+   *
+   * @param minuend - set being subtracted from
```

Comment:
it's all greek to me. lmk if there's a better word.

Reply:
do we need this method? we can just use `Sets.difference` in guava

Reply:
haha. didn't know that existed. guava does it again. i will remove this.